src/OrchardCore.Modules/OrchardCore.Twitter/AdminMenu.cs

```diff
@@ -18,17 +18,28 @@ public AdminMenuSignin(IStringLocalizer<AdminMenuSignin> localizer)
 
```

Comment:
Why this class and few other are decorated with `[Feature]` attribute (i.e., `[Feature(TwitterConstants.Features.Signin)]`?) Each of these classes get registered only when the corresponding feature is enabled (in this case "OrchardCore.Twitter.Signin"). I think adding the `[Feature]` attribute here is redundant. `[Feature]` attribute is helpful in some scenarios like a controller, but not sure how it is helpful here?

Reply:
Question for @jtkech
I think these are redundant too. Though, maybe there is a reason.